core/components/organisms/table/Table.tsx

```diff
@@ -609,13 +620,26 @@ export class Table extends React.Component<TableProps, TableState> {
 
     const indexes = Array.from({ length: data.length }, (_, i) => i);
 
-    const newData = updateBatchData(data, indexes, {
-      _selected: selected,
+    const newData = updateBatchData(
+      data,
+      indexes,
+      {
+        _selected: selected,
+      },
+      this.props.selectDisabledRow
+    );
+
+    const selectedIndex: number[] = [];
+
+    newData.forEach((item, key) => {
```

Comment:
You can use filter here instead of forEach to get the array of selected items.

Reply:
Here we need to get the **index** of selected items instead of the selected items itself